src/store/reducers/forging.js

```diff
@@ -0,0 +1,34 @@
+import actionTypes from '../../constants/actions';
+
+/**
+ *
+ * @param {Array} state
+ * @param {Object} action
+ */
+const forging = (state = { forgedBlocks: [], statistics: {} }, action) => {
+  let startTimesamp;
+  let endTimesamp;
+
+  switch (action.type) {
+    case actionTypes.forgedBlocksUpdated:
+      startTimesamp = state.forgedBlocks.length ? state.forgedBlocks[0].timestamp : 0;
+      endTimesamp = state.forgedBlocks.length ?
+        state.forgedBlocks[state.forgedBlocks.length - 1].timestamp :
+        0;
+      return Object.assign({}, state, {
+        forgedBlocks: [].concat(
```

Comment:
Replace [].concat with es6 spread syntax. You can find more about spread syntax here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator